ResearchKit/ActiveTasks/ORKTowerOfHanoiStep.m

```diff
@@ -77,4 +112,4 @@ - (BOOL)shouldContinueOnFinish {
     return YES;
 }
 
-@end
+@end
```

Comment:
Sorry for being nit-picky after the merge, but all the rest of source files end with a blank newline.